GDJS/Runtime/websocket-debugger-client/websocket-debugger-client.js

```diff
@@ -384,6 +381,7 @@ gdjs.WebsocketDebuggerClient.prototype._depthLimitedSerializer = function(
       if (maxDepth != null && thisPos > maxDepth) {
         return '[Max depth reached]';
       } else if (~stack.indexOf(value))
+      // @ts-ignore
```

Comment:
That is ignored as it says "cycleRepeater might be undefined", but as you can see above it is checked for undefined and null and if it is one of both it get defined.